src/position.service.js

```diff
@@ -149,12 +149,12 @@
      * @param containerWidth: the width of the dynamic-layout container
      * @return: the promise of the position animations being completed
      */
-    function layout(containerWidth) {
+    function layout(containerWidth, colWidth) {
       // We first gather the items dimension based on the DOM elements
       items = self.getItemsDimensionFromDOM();
 
       // Then we get the column size base the elements minimum width
-      var colSize = getColSize();
+      var colSize = getColSize(colWidth);
```

Comment:
You can use
```
var colSize = colWidth || getColSize()
```
And leave `getColSize` as it were.

Also could you change `colSize` into `colWidth` and `getColSize` into `getColWidth` everywhere to be consistent?